src/theme.js

```diff
@@ -0,0 +1,36 @@
+import { createTheme } from '@mui/material/styles';
+
+const theme = createTheme({
+  typography: {
+    fontFamily: ['-apple-system', 'BlinkMacSystemFont', '"Segoe UI"', 'Roboto'].join(',')
```

Comment:
Just want to make sure. Is "-apple-system", "BlinkMacSystemFont", and "Segoe UI" going to be used for styling fonts in PASS? If it is, I think it's fine. But if they're not, perhaps we can cut down on them to just "Roboto"

Reply:
That decision will probably be ultimately left to UX/UI. However, cutting down so much does change how it has looked since we first started using it

Reply:
Avoid resolving conversations that was started from others to avoid confusion.

Reply:
Agree to remove extra fonts for now leaving just 'Roboto'. We can add future ones when the need arises. Otherwise I'm good with this merging to Development.

Reply:
Let's trim them down. Jared makes a good point. We'll cross that bridge when when the UX/UI team decides we actually need more fonts for PASS.